### ISSUE TITLE
MUNIT

### ISSUE THREAD

**Issue by gcosne** [cc-ai/kdb#84]
Training [MUNIT](https://arxiv.org/abs/1804.04732), from [NVLabs](https://github.com/NVlabs/MUNIT):

After 250K epochs on a dataset of 1500 images from validation set of mapillary and [700 flood imgs](https://docs.google.com/document/d/1HhsSUkV8P5VktfYg6bAT31c6LfafDCf9Z8nEINNq_S4/edit?usp=sharing) automatically segmented.

**Randomly generated style:**
![alt-text-1](https://user-images.githubusercontent.com/25148268/60818337-7d767a80-a16b-11e9-91f9-6be825aa529f.png "title-1") ![alt-text-2](https://user-images.githubusercontent.com/25148268/60818370-88310f80-a16b-11e9-8e24-e147f9c0fc34.png "title-2")

I have an issue using style guided translation : 
**Guided Style translation:**
![alt-text-2](https://user-images.githubusercontent.com/25148268/60820036-a0eef480-a16e-11e9-98a8-c0a9b8b66775.png "title-2")![alt-text-1](https://user-images.githubusercontent.com/25148268/60820032-9fbdc780-a16e-11e9-924e-bf225d46be11.png "title-1") 

cf: https://github.com/NVlabs/MUNIT/issues/59

**Comment by gcosne** [comment]
In https://github.com/NVlabs/MUNIT/issues/27, the author states that enabling explicit cycle consistency loss, help the model converges faster and better. (I had this parameter set to 0 in the previous experiment).
